tests/languages/elisp/quoted-symbol_feature.test

```diff
@@ -0,0 +1,17 @@
+(

```

Comment:
I was unsure about how to name these files where the feature is hyphenated. Let me know if it should be changed

Reply:
This is perfectly fine. :)